amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -108,20 +108,18 @@ Anypoint Connector for Amazon SQS (Amazon SQS Connector) enables you to interfac
 === Add Permission
 `<sqs:add-permission>`
 
-
 This operation adds a permission to a message queue.
 
-
 ==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | x
+| Configuration | String | Name of the configuration to use | | x
 | Label a| String |  Name for this permission |  | x
 | Account Ids a| Array of String |  IDs of the AWS accounts to share this queue with |  | x
 | Actions a| Array of String |  List to indicate how much to share (SendMessage, ReceiveMessage, ChangeMessageVisibility, DeleteMessage, GetQueueAttributes) |  | x
-| Queue Url a| String |  Permissions will be added to the queue represented by this URL. This parameter is optional and if you do not specify queueUrl you need to set in the configuration level Default Global Queue URL. |  | 
-| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Queue Url a| String |  Permissions will be added to the queue represented by this URL. This parameter is optional, and if you do not specify `queueUrl`, you must specify the Default Global Queue URL at the configuration level. |  | 
+| Target Variable a| String | Name of a variable in which to store the operation's output |  | 
```

Comment:
ditto

Reply:
I'm going to stop marking these; please give all items in one column a period if even one of them ends with on.
Same with items in a list.